mission_control/navigator_missions/navigator_missions/entrance_gate.py

```diff
@@ -0,0 +1,463 @@
+#!/usr/bin/env python
+from txros import util
+from navigator_missions.navigator import Navigator
+import numpy as np
+from mil_tools import rosmsg_to_numpy
+from twisted.internet import defer
+import math
+from mil_misc_tools import ThrowingArgumentParser
+from argparse import ArgumentTypeError
+import tf2_ros
+import rospy
+
+
+class EntranceGate(Navigator):
+    @classmethod
+    def decode_parameters(cls, parameters):
+        argv = parameters.split()
+        return cls.parser.parse_args(argv)
+
+    @classmethod
+    def init(cls):
+        def str2bool(v):
+            '''
+                https://stackoverflow.com/a/43357954
+            '''
+            if v.lower() in ('yes', 'true', 't', 'y', '1'):
+                return True
+            elif v.lower() in ('no', 'false', 'f', 'n', '0'):
+                return False
+            else:
+                raise ArgumentTypeError('Boolean value expected.')
+
+        class Range(object):
+            '''
+                https://stackoverflow.com/a/12117089
+            '''
+            def __init__(self, start, end):
+                self.start = start
+                self.end = end
+
+            def __eq__(self, other):
+                return self.start <= other <= self.end
+
+        parser = ThrowingArgumentParser(description='Start Gate Mission',
+                                        usage='''Default parameters: \'runtask EntranceGate --passscan false
+                                         --multilateration false --scandist 10 --speed 0.75 --kill true
+                                         --scantime 10 --traversaldist 7\'''')
+        parser.add_argument('-p', '--passscan', type=str2bool, default=False,
```

Comment:
More conventional to use ```type='set_true'``` like [here](https://github.com/uf-mil/NaviGator/blob/c3408b30b5ad4a62211f4fb056efc0b9be738519/mission_control/navigator_missions/navigator_missions/move.py#L32). So you would just do `... -m` for multiliteration rather than `... -m true`

Reply:
Fixed https://github.com/jaxnb/NaviGator/commit/72279f71d82bbfb332a10b010744dba891e938bc